src/logic/utils.ts

```diff
@@ -44,10 +44,10 @@ export function getAverageWinTurn(player: Player) {
 export function isNewlyQualifiedPlayer(player: Player) {
     const dateOffset = new Date();
     dateOffset.setDate(dateOffset.getDate() - NEW_PLAYER_HIGHLIGHT_DAYS);
-    const lastIndex = player.matches.length - 1;
+    const lastIndex = player.validMatchesCount - 1;
     if (
-        PLAYER_MINIMUM_GAMES_REQUIRED <= player.matches.length &&
-        player.matches.length <= 15 &&
+        PLAYER_MINIMUM_GAMES_REQUIRED <= player.validMatchesCount &&
```

Comment:
this is an interesting change-- i don't think this applies to anyone, but does this actually "unqualify" any players? :O

Reply:
It does not unqualify anyone, I think it's just better to do this now rather than wonder later why there's an anomaly in displayed data. If left the way it was we'd see a player qualify with fewer than 10 matches because the match counter in the column refers to validMatches anyway.

Say testPlayerPeter tags along to the server and plays five 3-player games and five 4-player games. We'd see them as qualified but with only five matches which would be super weird. The edge case is actually a very visible one in case it happens because the player is new and has very few matches at this point. Even one non-4-player-game would throw this off.